lhotse/dataset/collation.py

```diff
@@ -1,11 +1,109 @@
-from typing import Iterable, Union
+from typing import Iterable, Union, Literal, Tuple, Callable
+from dataclasses import dataclass
 
 import numpy as np
 import torch
 from torch.nn import CrossEntropyLoss
 
 from lhotse import CutSet
 from lhotse.cut import MixedCut
+from lhotse.utils import LOG_EPSILON
+
+
+@dataclass
+class CollateTokens:
+
+    tokens: Iterable[str]
```

Comment:
I think this + post init is a hack in this case, because we don't need the original iterable after initialization. I think this should be a regular class with init that either accepts an iterable or a previously prepared mapping (or one of both)

Reply:
Fair point, I was a bit lazy with making regular init. Will change it to regular classes :)